draft/2025-10-15-this-week-in-rust.md

```diff
@@ -122,7 +124,11 @@ Some of these tasks may also have mentors available, visit the task page for mor
 <!-- * [ - ]() -->
 <!-- or if none - *No Calls for participation were submitted this week.* -->
 
-If you are a Rust project owner and are looking for contributors, please submit tasks [here][guidelines] or through a [PR to TWiR](https://github.com/rust-lang/this-week-in-rust) or by reaching out on [X (formerly Twitter)](https://x.com/ThisWeekInRust) or [Mastodon](https://mastodon.social/@thisweekinrust)!
```

Comment:
If there is a problem with the boilerplate text, please open a separate issue for that. Changes like this should not be included in the same PR as other contributions.